docs/development.md

```diff
@@ -25,7 +25,6 @@ Edit values in `.env` to point to existing postgresql databases. See `nmdc_serve
 ```bash
 NMDC_CLIENT_ID=changeme
 NMDC_CLIENT_SECRET=changeme
```

Comment:
These should be changed to NMDC_ORCID_CLIENT_ID right?

Reply:
Yep! Good catch